{
  "cveId": "CVE-2024-31881",
  "title": "IBM Db2 denial of service",
  "description": "IBM Db2 for Linux, UNIX and Windows (includes Db2 Connect Server) 10.5, 11.1, and 11.5 is vulnerable to a denial of service as the server may crash when using a specially crafted query on certain columnar tables by an authenticated user.  IBM X-Force ID:  287613.",
  "publishedDate": "2024-06-12T18:21:45.458000+00:00",
  "lastModifiedDate": "2024-08-02T01:59:50.628000+00:00",
  "severity": "MEDIUM",
  "cvssMetrics": [
    {
      "version": "3.1",
      "vectorString": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H",
      "baseScore": 6.5,
      "baseSeverity": "MEDIUM"
    }
  ],
  "epss": {
    "score": 0.0019,
    "percentile": 41.336
  },
  "riskScore": 40,
  "exploitationStatus": "UNKNOWN",
  "affectedSystems": {
    "cpeMatches": [],
    "vendors": ["ibm"],
    "products": ["db2 for linux, unix and windows"]
  },
  "references": [
    {
      "url": "https://www.ibm.com/support/pages/node/7156852",
      "source": "",
      "tags": ["vendor-advisory"]
    },
    {
      "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/287613",
      "source": "",
      "tags": ["vdb-entry"]
    }
  ],
  "attackTechniques": [],
  "metadata": {
    "tags": ["CWE-770"],
    "requiresUserInteraction": null,
    "requiresPrivileges": null,
    "attackVector": null,
    "sources": [
      {
        "name": "CVEList",
        "url": "https://github.com/CVEProject/cvelistV5/blob/main/cves/2024/31xxx/CVE-2024-31881.json",
        "lastModified": "2024-08-02T01:59:50.628000+00:00"
      }
    ]
  }
}
